docs/source/reference/cuda.rst

```diff
@@ -21,12 +21,16 @@ Memory management
    cupy.get_default_memory_pool
    cupy.get_default_pinned_memory_pool
    cupy.cuda.Memory
+   cupy.cuda.PinnedMemory
    cupy.cuda.MemoryPointer
+   cupy.cuda.PinnedMemoryPointer
    cupy.cuda.alloc
+   cupy.cuda.alloc_pinned_memory
    cupy.cuda.set_allocator
    cupy.cuda.set_pinned_memory_allocator
    cupy.cuda.MemoryPool
    cupy.cuda.PinnedMemoryPool
+   cupy.cuda.SingleDeviceMemoryPool
```

Comment:
ditto.

Reply:
Do you want to hide it on the document?
It is referred from `cupy.cuda.MemoryHook` now.

Which do you think better?

- Remove reference to `SingleDeviceMemoryPool`.
- Refer to `cupy.cuda.memory.SingleDeviceMemoryPool`.

In second option we can hide it on the list, it is just referred from `MemoryHook`.

Reply:
OK, Cloud you fix `MemoryHook` document?
I think that the link to `MemoryPool` is enough for that explanation.

Reply:
OK, I skip it here, and make another PR to fix `MemoryHook` document.
I want to treat only easy reference errors in this PR.